rules/models/changes/ruleset.go

```diff
@@ -0,0 +1,96 @@
+package changes
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+)
+
+// UpdateOp is an update operation.
+type UpdateOp int
+
+// A list of supported update operations.
+const (
+	UnknownOp UpdateOp = iota
```

Comment:
Would it be easier if you do
```
type Op string

const (
  AddOp Op       = "add"
  ChangeOp Op = "change"
  DeleteOp Op   = "delete"
)
```
so you don't need to do manually parsing and marshaling/unmarshaling etc?

Reply:
I can make this change. I dont think this will change the ordering of the different sorting schemes as they are already in alphabetical order, but ill have to take a look at that.

Reply:
Yeah that should work regardless, we don't necessarily need them to be ordered as additions first, then changes, then deletions as long as the operations for the same op type are grouped together.